### PR TITLE
Fixing up lots of summaries

### PR DESCRIPTION
**Pull request by Thundermaker300**
* Essentially doing the devil's work
  - Fixing up multiple typos (including a typo that said SCP-096 creates portals, duplicated "mm" in an ammunition enum, etc)
  - All event handlers' summaries now say "Invoked ..." while all of their respective methods say "Called ..."
  - Adding consistency between event arguments and their respective handlers
  - Adding consistent usage of "SCPs", "SCP-###", etc.
  - Fixing miscellaneous English grammatical errors
* If you review these manually: Sorry in advance, I felt really dedicated tonight

### CODE REVIEW

File: Exiled.API/Features/Cassie.cs [hunk 1]
```diff
@@ -17,7 +17,7 @@ namespace Exiled.API.Features
     public static class Cassie
     {
         /// <summary>
-        /// Gets a value indicating whether or not a C.A.S.S.I.E announcement is currently playing. Does not include decontamination messages.
+        /// Gets a value indicating whether or not a C.A.S.S.I.E announcement is currently playing or if a C.A.S.S.I.E announcement is queued. Does not include decontamination messages.
```
Comment (iRebbok) [inline Exiled.API/Features/Cassie.cs:20]: But really, when you queue a CASSIE announcement, it returns false. That's the correct summary, undo the change.

**Review by iRebbok — APPROVED** [review]

**Review by iRebbok — APPROVED** [review]

File: Exiled.Events/EventArgs/PickingUpScp330EventArgs.cs [hunk 2]
```diff
@@ -33,7 +33,7 @@ public PickingUpScp330EventArgs(Player player, int usage, ItemType item, bool is
         }
 
         /// <summary>
-        /// Gets the player who picking up the scp330.
+        /// Gets the player who picking up SCP-330.
```
Comment (iopietro) [inline Exiled.Events/EventArgs/PickingUpScp330EventArgs.cs:36]: Another one: `who's picking up`

File: Exiled.Events/EventArgs/KickedEventArgs.cs [hunk 3]
```diff
@@ -12,7 +12,7 @@ namespace Exiled.Events.EventArgs
     using Exiled.API.Features;
 
     /// <summary>
-    /// Contains all iformations after banning a player from the server.
+    /// Contains all iformations after kicking a player from the server.
```
Comment (iopietro) [inline Exiled.Events/EventArgs/KickedEventArgs.cs:15]: I found another typo: `informations`

File: Exiled.Events/Handlers/Player.cs [hunk 4]
```diff
@@ -495,85 +498,85 @@ public static class Player
...
         /// <summary>
-        /// Called before changing user mute status.
+        /// Called before a user's mute status is changed.
         /// </summary>
         /// <param name="ev">The <see cref="ChangingMuteStatusEventArgs"/> instance.</param>
         public static void OnChangingMuteStatus(ChangingMuteStatusEventArgs ev) => ChangingMuteStatus.InvokeSafely(ev);
 
         /// <summary>
-        /// Called before changing user intercom mute status.
+        /// Called before a user's intercom mute status is changed.
```
Comment (iopietro) [inline Exiled.Events/Handlers/Player.cs:579]: Another one: `an user's`

File: Exiled.Events/Handlers/Player.cs [hunk 5]
```diff
@@ -495,85 +498,85 @@ public static class Player
...
         /// <summary>
-        /// Called before deactivating workstation.
+        /// Called before a workstation is deactivated.
         /// </summary>
         /// <param name="ev">The <see cref="DeactivatingWorkstationEventArgs"/> instance.</param>
         public static void OnDeactivatingWorkstation(DeactivatingWorkstationEventArgs ev) => DeactivatingWorkstation.InvokeSafely(ev);
 
         /// <summary>
-        /// Called before changing user mute status.
+        /// Called before a user's mute status is changed.
```
Comment (iopietro) [inline Exiled.Events/Handlers/Player.cs:573]: Another one: `an user's`

File: Exiled.Events/Handlers/Player.cs [hunk 6]
```diff
@@ -203,287 +206,287 @@ public static class Player
...
 
         /// <summary>
-        /// Invoked before changing user mute status.
+        /// Invoked before a user's mute status is changed.
         /// </summary>
         public static event CustomEventHandler<ChangingMuteStatusEventArgs> ChangingMuteStatus;
 
         /// <summary>
-        /// Invoked before changing user intercom mute status.
+        /// Invoked before a user's intercom mute status is changed.
```
Comment (iopietro) [inline Exiled.Events/Handlers/Player.cs:274]: `an user's`
